docs/development-api/github.md

```diff
@@ -0,0 +1,41 @@
+---
+parent: Development API
+---
+
+# GitHub
+
+---
+
+## createCheckRun
```

Comment:
This has me a bit conflicted; while this is the simplest API to support typical SEEK usage, I wonder if it's too Buildkite-specific for the name.

What do you think about:

1. Having a more wordy name like `createCheckRunFromBuildkite` to qualify that this is tailored towards Buildkite usage

2. Accepting an optional `auth` or `githubApiToken` parameter that falls back to the environment variable, so it still works out of the box for us but isn't intrinsically tied to SEEK's custom `GITHUB_API_TOKEN`?

   (I haven't actually convinced myself that this is a good idea, and maybe we can leave it until someone asks given it would be a non-breaking addition.)

Reply:
1. Happy to use `createCheckRunFromBuildkite` given we are relying on `BUILDKITE_` environment variables.

2. 🤔 Hmmm I'm going to lean towards leaving it til someone requests it.